dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/navigation/NavTool.java

```diff
@@ -77,13 +79,24 @@ protected NavResultHydrated getNav(Host host, String path, long languageId, User
             path = path.substring(0, path.lastIndexOf("/"));
         }
 
-        Folder folder = !path.equals("/") ? APILocator.getFolderAPI()
+        final Folder originalFolder = !path.equals("/") ? APILocator.getFolderAPI()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Position literals first in String comparisons](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5378894)